app/scripts/controllers/deploymentConfig.js

```diff
@@ -63,6 +63,17 @@ angular.module('openshiftConsole')
     });
     AlertMessageService.clearAlerts();
 
+    // copy deploymentConfig and ensure it has env so that we can edit env vars using key-value-editor
+    var copyDeploymentConfigAndEnsureEnv = function(deploymentConfig) {
+      $scope.updatedDeploymentConfig = angular.copy(deploymentConfig);
+      $scope.updatedDeploymentConfig.spec.template.spec.containers = _.map($scope.updatedDeploymentConfig.spec.template.spec.containers, function(container){
```

Comment:
The `_.each` version is definitely shorter!  I think this was just a decision of favoring `map` over `each`, but I agree, each is a good call here.